cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -14,17 +14,19 @@
  * the License.
  */
 
-function LogViewerController ($scope, LogViewerStore, myLogsApi, LOGVIEWERSTORE_ACTIONS, MyCDAPDataSource, $sce, myCdapUrl, $timeout) {
+function LogViewerController ($scope, LogViewerStore, myLogsApi, LOGVIEWERSTORE_ACTIONS, MyCDAPDataSource, $sce, myCdapUrl, $timeout, $uibModal) {
   'ngInject';
 
   var dataSrc = new MyCDAPDataSource($scope);
   var pollPromise;
   //Collapsing LogViewer Table Columns
   var columnsList = [];
   var collapseCount = 0;
+  this.$uibModal = $uibModal;
 
   this.setDefault = () => {
     this.textFile = null;
+    this.statusType = 0;
```

Comment:
if you are setting the program status default as 'Not Started', this statusType should be 3. Otherwise it will say status Not Started but the circle is green